### DISCUSSION TITLE
snapshots

### DISCUSSION THREAD

**Discussion by cocampbe** [longhorn/longhorn#4088] [Q&A]
longhorn version 1.2.3. I see snapshots listed under "Snapshots and Backups". I am wondering what kubectl command I can run to see these snapshots. I tried looking at the volumesnapshots CRD and others, but cannot find them. I was wondering if someone could point me in the right direction.

**Comment by cocampbe** [comment]
Another question I have is, what is the difference between green and blue snapshots. I was under the impression that green was a snapshot and blue was a backup. But I have seen some green icons with the word backup in the name.

![image](https://user-images.githubusercontent.com/2211329/172392074-72c86b63-f60e-4379-b15e-fcdec7932f38.png)

**Comment by derekbit** [comment]
> longhorn version 1.2.3. I see snapshots listed under "Snapshots and Backups". I am wondering what kubectl command I can run to see these snapshots. I tried looking at the volumesnapshots CRD and others, but cannot find them. I was wondering if someone could point me in the right direction.

You can see the snapshots in the `engine` resources in v1.2.x.
